src/sensors/simulator_bridge.cpp

```diff
@@ -0,0 +1,131 @@
+#include "ros/ros.h"
+#include "std_msgs/Float32.h"
+#include "geometry_msgs/Quaternion.h"
+#include "geometry_msgs/Vector3.h"
+#include "gazebo_msgs/ModelState.h"
+#include "gazebo_msgs/ModelStates.h"
+#include "robosub/ObstaclePosArray.h"
+
+ros::Publisher position_pub;
+ros::Publisher orientation_pub;
+ros::Publisher depth_pub;
+ros::Publisher obstacle_pos_pub;
+//ros::Publisher lin_accel_pub;
```

Comment:
Remove commented out lines of code  if you plan on adding them in future pull requests.